### DISCUSSION TITLE
Azure communication chat implementation

### DISCUSSION THREAD

**Discussion by jonasboardwalker** [chatscope/use-chat#4] [Q&A]
## Thank you
First I'd like to say thanks to @supersnager for an amazing library (definitely more than just a hook)!

## What I would like to know
How to integrate Azure communication chat service to my web application using useChat hook

**Comment by supersnager** [comment]
@jonasboardwalker I'glad you like this project :)
Unfortunately, I have no experience with Azure communication, however I just started writing a tutorial on how to use the library with a simple nodejs communication server. It will take some time to write, but I think it will be helpful to understand basic operation principles. Connecting other services should be similar.

**Reply by jonasboardwalker** [reply]
Thank you so much for the response! Tutorial would be very helpful

**Reply by supersnager** [reply]
BTW, I'm considering making the adapter for Azure communications, but after that basic tutorial is done.
